src/main/target/system_stm32f7xx.c

```diff
@@ -242,6 +242,20 @@
           while (1);
       }
 
+#ifdef USEMCO2ASOSDCLOCK
+      PeriphClkInitStruct.PeriphClockSelection = RCC_PERIPHCLK_PLLI2S;
+      PeriphClkInitStruct.PLLI2S.PLLI2SN = 216;
```

Comment:
This doesn't take F7 Overclock facility into account?

Reply:
I didn't know overclocking F7 was a thing. How high can it go?

Reply:
Two options: 216 (off) and 240 MHz.
Just check `target/system_stm32f7xx.c`

Reply:
If I understand the code correctly the PLLM output will be the same regardless of which clock is used and the overclock is applied after that.
So it should work with and without OC